components/fxa-client/src/fxa_client.udl

```diff
@@ -723,6 +709,27 @@ interface FirefoxAccount {
   
 };
 
+dictionary FxaConfig {
+    // FxaServer to connect with
+    FxaServer server;
+    // Registered OAuth client id of the application.
+    string client_id;
+    // `redirect_uri` - the registered OAuth redirect URI of the application.
+    string redirect_uri;
+    //  URL for the user's Sync Tokenserver. This can be used to support users who self-host their
+    //  sync data. If `None` then it will default to the Mozilla-hosted Sync server.
+    string? token_server_url_override;
```

Comment:
I think the fact that this exists at all is suspect tbh - I don't see why this module needs to know about the override - but that existed well before this :)

Reply:
Probably for pre-sync manager implementations, we might have an API that retrieves the token server URL for consumers

Reply:
yeah, that might even be true today, but it still seems wrong for this to know about the override - this crate should probably just return what fxa told us, and the choice about whether to use the override made by the consumer of this